### ISSUE TITLE
Flyers first pass check

### ISSUE THREAD

**Issue by IJW-Wartrader** [GameBrains/er-core#31]
I'm about to start checking through the Flyers section, and have a few requests:

- All the missions contain an identical 'Declare the mission' section. Can I move this to flyer-missions.adoc?
- You've put the missions in alphabetical order, but this means that the Air Transport mission refers to the Ground Attack mission which comes _afterwards_. Can I change the order of the missions back to the original Ground Attack/Air Transport/Intercept sequence?

**Comment by IJW-Wartrader** [comment]
On a separate topic, I'm currently working on revised text for all references to 'hits' on Flyers, as the original GW was horribly ambiguous – as you've found yourself and as I've observed in several discussions in person and online.

I'm probably going to borrow the 'successful hit dice' terminology from the Shooting phase, and reserve 'hit' for successfully making the second Armour roll.

In _very_ rough form, something like this:

1. Make your Snap Fire rolls.
2. Discard dice that scored lower than the lowest Armour value.
3. Allocate successful hit dice.
4. Discard any allocated dice lower that Flyer's Armour.
5. Roll any remaining dice to see if the Flyer is driven off (and temporarily damaged), or if it is hit.

For dogfights it gets simplified to:

1. Make your Gunnery dice rolls against a target.
2. Discard dice that are lower than the target's Armour.
3. Roll any remaining dice to see if the Flyer is driven off (and temporarily damaged), or if it is hit.

**Comment by digi-brain** [comment]
> On a separate topic, I'm currently working on revised text for all references to 'hits' on Flyers, as the original GW was horribly ambiguous – as you've found yourself and as I've observed in several discussions in person and online.

Yes indeed, horribly ambiguous at times. What you suggest sounds good on the whole to me. When we get into the nitty gritty we might debate some specific terminology, but I agree entirely with the overall idea that we should carry through on our efforts to rationalise and disambiguate terminology around 'hits' and fix the issues in the flyers rules.

**Comment by digi-brain** [comment]
I started to try to reply to your points, but I think I simply don't see what you're getting at. So yes, I think a video call might be a simpler way to go. Or indeed you could do one to demonstrate and perhaps it'll become clearer. Happy to do either/both as you see fit.

---

As an aside to the main issue, a more general question that this throws up for me:

> *I've already corrected the readiness states file to say that you need the HQ unit in Ready state, rather than a regular Flyer unit.

I wonder how meaningful the HQ/command rules are for flyers? I know you have to pay the Detachment HQ cost and they have a chain of command — but what do the HQ rules actually do for flyers, aside from implying a limit to how far you can spread them out for their attack run? If the HQ specifically must be Ready before you can declare a mission, doesn't that make flyers even more crap in this edition? The entire rest of the detachment could be Ready but you're stuck for another turn because the HQ is in the Rearm state?

---

>Flyers and Flak – I'm currently planning to rewrite the Flak orders section so that it uses multiple Snap Fire rolls, rather than GW's approach of using completely new text for something that (at the per-die level) is functionally identical to Snap Fire.

I don't see exactly what you mean, but I think I get the general idea. Even after I'd reworked the original text, it still felt clumsy somehow. If you have a way to rework the rules to something functionally equivalent but simpler or more consistent then that seems like a good thing!

**Comment by IJW-Wartrader** [comment]
Doh, I thought I'd posted this reply, but apparently not...

> As an aside to the main issue, a more general question that this throws up for me:
> 
> > *I've already corrected the readiness states file to say that you need the HQ unit in Ready state, rather than a regular Flyer unit.
> 
> I wonder how meaningful the HQ/command rules are for flyers? I know you have to pay the Detachment HQ cost and they have a chain of command — but what do the HQ rules actually do for flyers, aside from implying a limit to how far you can spread them out for their attack run? If the HQ specifically must be Ready before you can declare a mission, doesn't that make flyers even more crap in this edition? The entire rest of the detachment could be Ready but you're stuck for another turn because the HQ is in the Rearm state?

As best as I can parse the Q&A for Flyer detachments that have got split, yes.

> Q: When members of a flyer detachment become damaged, that detachment has two elements that finish rearming/refueling on differing turns. Which element(s) of this flyer detachment may fly missions, and when may they fly them?
> 
> a. Both elements of the detachment may fly missions whenever they become rearmed and refueled. The element of the detachment that does not include an HQ should select one flyer to act as its HQ.
> b. Both elements. It is not necessary to choose an interim HQ for element that does not have one.
> c. Only the element of the detachment which includes the HQ may fly missions when rearmed and refueled. When the other element is rearmed/refueled, it must still wait to join its brethren with the HQ.
> d. Only the bigger element of the detachment may fly independent missions, regardless of which element the HQ is in.
> Neither element may fly a mission until the other is also rearmed and refuelled.
> 
> A: A flyer that is damaged can't fly missions on its own (unless it is all that remains of the detachment, of course!). If the rest of the detachment flies a mission while the damaged flyer is being refueled, it simply remains for an extra turn at the refueling stage in order to catch up with them.

From https://thehobby.zone/resources/e40k-compendium/Content/More/QuestionsAndAnswers/QA_Flyers.htm

----

> > Flyers and Flak – I'm currently planning to rewrite the Flak orders section so that it uses multiple Snap Fire rolls, rather than GW's approach of using completely new text for something that (at the per-die level) is functionally identical to Snap Fire.
> 
> I don't see exactly what you mean, but I think I get the general idea. Even after I'd reworked the original text, it still felt clumsy somehow. If you have a way to rework the rules to something functionally equivalent but simpler or more consistent then that seems like a good thing!

Roughly speaking:
Flak units get to Snap fire at full Range against Flyers and Drop Pods.
Flak units on Flak orders get a pool of Snap Fire shots to use this way, and can choose how many shots to use at once.

But I'll see how messy it gets as actual written rules...

**Comment by IJW-Wartrader** [comment]
I've committed the changes so far, including the Flak rewrite. You can have a look now if you want, or wait until I've rewritten one of the missions, and then I'll start a pull request.

**Comment by IJW-Wartrader** [comment]
Pull request is up: https://github.com/GameBrains/er-core/pull/33

**Comment by IJW-Wartrader** [comment]
As a heads-up, I'm going to start topics on FB about Flyers and HQs.

Locally, we've found that targeting the HQ during an intercept often ends up making the whole detachment ineffective for the rest of the game (unless it's a very long game) as the rest of the detachment has to sit there and wait until the HQ has been repaired.

**Comment by digi-brain** [comment]
@IJW-Wartrader Thanks, good to know. All told, it would seem that what with one thing and another the original aircraft rules are a bit naff... 😂
